pkgs/development/libraries/gegl/3.0.nix

```diff
@@ -3,21 +3,27 @@
 , libwebp, gnome3 }:
 
 stdenv.mkDerivation rec {
-  name = "gegl-0.3.26";
+  name = "gegl-0.3.28";
 
   src = fetchurl {
     url = "http://download.gimp.org/pub/gegl/0.3/${name}.tar.bz2";
-    sha256 = "1a9zbi6ws0r0sqynvg2fh3ad0ipnphg7w62y7whlcrbpqi29izvf";
+    sha256 = "1zr3gmmzjhp2d3d3h51x80r5q7gs9rv67ywx69sif6as99h8fbqm";
   };
 
+  NIX_LDFLAGS = stdenv.lib.optionalString stdenv.isDarwin "-lintl";
+
   hardeningDisable = [ "format" ];
 
   # needs fonts otherwise  don't know how to pass them
   configureFlags = "--disable-docs";
 
   enableParallelBuilding = true;
 
-  doCheck = true;
+  # GEGL_SWAP=RAM GEGL_PATH=../../operations ./buffer-test
+  # dyld: Library not loaded: @rpath/libjson-glib-1.0.0.dylib
+  #   Referenced from: /private/tmp/nix-build-gegl-0.3.28.drv-0/gegl-0.3.28/tests/buffer/.libs/buffer-test
+  #   Reason: image not found
+  doCheck = !stdenv.isDarwin;
```

Comment:
I think dependencies that link against this library will run into the same issue.

Reply:
What could be causing it? We should probably fix it before GIMP 2.10 is released.